tests/settings/settings.go

```diff
@@ -0,0 +1,33 @@
+package settings
+
+import (
+	"os"
+	"time"
+)
+
+var (
+	ActualHome               = os.Getenv("HOME")
+	TestHome                 string
+	TestRoot                 string
+	DeisControllerURL        = os.Getenv("DEIS_CONTROLLER_URL")
```

Comment:
Can you please change this back to `deis.$DEIS_ROUTER_SERVICE_HOST:$DEIS_ROUTER_SERVICE_PORT.nip.io`?